### ISSUE TITLE
CoreAPI throws a TypeError when contentType is null

### ISSUE THREAD

**Issue by bluesurfer** [encode/django-rest-framework#5059]
When the API returns no response (like after a delete request) the coreapi javascript client library throws a `TypeError: Cannot read property 'toLowerCase' of null`. The problem is in `utils.js`:
```
const negotiateDecoder = function (decoders, contentType) {
  if (contentType === undefined) {
    return decoders[0]
  }

const fullType = contentType.toLowerCase().split(';')[0].trim() // <- error here
```
We probably should add the case catching `contentType === null`?

**Comment by tomchristie** [comment]
Yup. Also related is https://github.com/core-api/javascript-client/pull/35

**Comment by bluesurfer** [comment]
Why is this taking so long to be fixed? This is a pretty critical bug.

In the meantime, I am using this workaround:

```
class BookViewSet(ModelViewSet):
    ...
    def destroy(self, request, *args, **kwargs):
        super().destroy(request, *args, **kwargs)
        return Response({'message': 'book deleted'}, status=HTTP_200_OK)
```

**Comment by tomchristie** [comment]
Lots of different spinning plates. I'll move back to ticket resolution thurs/fri.

**Comment by tomchristie** [comment]
Closed via #37.

**Comment by tomchristie** [comment]
Will be resolved in the 3.7 release, upcoming shortly.
